### PR TITLE
Add: tqdm

### PR DESCRIPTION
**Pull request by casperdcl**
**GH Stars:** 18.3 k (https://github.com/tqdm/tqdm)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
  - Source: @tqdm maintainer (@casperdcl) -> Blender -> PNG (1024x1024) -> Inkscape trace bitmap -> https://img.tqdm.ml/logo.svg
  - the primary hex value is `2196F3` while the secondary is `FFC107`. Which do you think looks better?

**Primary:**
tqdm-2196F3

**Secondary:**
tqdm-FFC107

**Original:**
tqdm

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for contributing this one @casperdcl - the project is well within scope based on the GitHub stars alone. Looking at your path, given it's a bitmap trace there are a few excess points which can be cleared up by rebuilding the icon in Inkscape just using simple shapes and rounding the corners. I was able to generate [this path](https://user-images.githubusercontent.com/1830867/118321636-c5981e00-b4f5-11eb-8f66-9a7571f37239.png) with generally quite clean paths when [compared to the bitmap trace](https://user-images.githubusercontent.com/1830867/118321712-e3fe1980-b4f5-11eb-8867-50a47f27aedc.png). I've provided my path below, which isn't a perfect match - but you're welcome to use if you think it's a suitable alternative. Otherwise, you should be able to achieve similar results on your end by using the simple shapes within your vector program.

```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>tqdm icon</title><path d="M12 14.562a2.338 2.338 0 1 1 0-4.677 2.338 2.338 0 0 1 0 4.677zM12 0C5.392 0 .036 5.473.036 12.224c0 5.579 3.659 10.281 8.658 11.746.428.126.87-.162.962-.598l.141-.669c.086-.41-.169-.799-.57-.92-4.039-1.221-6.986-5.037-6.986-9.559 0-5.507 4.37-9.972 9.76-9.972s9.76 4.464 9.76 9.972c0 4.515-2.938 8.325-6.967 9.552-.4.122-.654.511-.567.919l.142.67c.093.437.535.723.963.596 4.986-1.474 8.633-6.169 8.633-11.738C23.964 5.473 18.608 0 12 0zm7.152 12.224c0-4.04-3.202-7.315-7.152-7.315s-7.152 3.275-7.152 7.315c0 3.191 1.999 5.903 4.786 6.902a.79.79 0 0 0 1.037-.582l.042-.199a.772.772 0 0 0-.489-.889c-2.118-.752-3.639-2.809-3.639-5.232 0-3.059 2.424-5.539 5.415-5.539s5.415 2.48 5.415 5.539c0 2.418-1.516 4.472-3.628 5.227a.772.772 0 0 0-.487.89l.042.199a.791.791 0 0 0 1.038.58c2.78-1.003 4.772-3.71 4.772-6.896z"/></svg>
```

**Comment by casperdcl** [comment]
Thanks, updated.

Also I think when monochrome, the secondary colour looks better so going to leave it as-is.

**Comment by adamrusted** [comment]
Amazing - I'll have to ping the other @simple-icons/maintainers to review this one, being as you've gone with my path. Hopefully someone can cast an eye over if ahead of this weekend's release! Thanks for this one @casperdcl 💪

**Comment by PeterShaggyNoble** [comment]
Thanks for the contribution, @casperdcl, and for the assistance, @adamrusted.

The only change that I'd suggest before merging is using [the commit URL](https://github.com/tqdm/img/blob/0dd23d9336af67976f88f9988ea660cde78c54d4/logo.svg) rather than the direct one for the source.

With regard to the colour, as a representative of the brand, we'll be guided by you, of course, but we would generally expect the primary colour to be used.

File: _data/simple-icons.json
Comment (casperdcl) [inline _data/simple-icons.json:9112]:
```suggestion
            "source": "https://github.com/tqdm/img/blob/0dd23d9336af67976f88f9988ea660cde78c54d4/logo.svg"
```

Comment (casperdcl) [reply _data/simple-icons.json:9112]: doesn't seem to be consistently used by other GH-hosted logos but whatever...

Comment (PeterShaggyNoble) [reply _data/simple-icons.json]: To clarify as to why I made that suggestion: with no website at http://tqdm.ml/, I'd be worried about the possibility of link rot in the future, making it harder for other contributors to find a suitable source when reviewing this one down the road.

**Review by PeterShaggyNoble — APPROVED** [review]
Anyway, this all looks good to me now, @casperdcl - thanks again for the contribution 👍🏻

**Pull request merged**
